website/content/docs/plugins/creation/plugin-faqs.mdx

```diff
@@ -0,0 +1,35 @@
+---
+description: |
+  Frequently asked questions for Packer Plugins.
+page_title: Packer Plugin FAQs - Extending
+---
+
+# Extending Packer Plugin FAQs
+
+1. **What are the responsibilities in maintaining a community plugin?**
+Community plugins are made available by the community, for the community to be used with Packer. By maintaining a plugin, you will be responsible for the development, release, and ongoing support of the plugin.
+The responsibilities of development include:
+  1. Updating the code base to work with new versions of Packer.
+  1. Reviewing issues and pull-requests opened against the plugin repository in a timely manner.
+  1. Releasing new versions of the plugin binary that are compatible with Packer - See [Creating a Release](https://www.packer.io/docs/plugins/creation#creating-a-github-release) for more details.
+
+2. **What happens to an active plugin that is no longer being maintained?**
+Plugins that are not actively maintained will be marked as unmaintained on their respective documentation pages, and scheduled for archival. We will try to find a new owner. If no owner can be found then HashiCorp will follow the process for archiving a plugin that has no owner.
+
+3. **What does it mean for a plugin to be archived?**
+A plugin that does not have an owner will be broken out into its own repository under the HashiCorp GitHub organization. The new repository will then be [archived](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/archiving-a-github-repository) to let users know that the plugin is no longer being maintained. Upon the initial breakout, a final compatible release of the plugin will be made, but it will not be actively maintained or monitored for new issues. Critical security-related issues will be reviewed and addressed where possible.
```

Comment:
this "breakout" has ultimately already happened, so feels confusing to list here.